MetaMask/metamask-extension
# Fix/14302 about fox logo

## Explanation
Updates the logo jpg images to use svg so they work in both light and dark modes

Fixes https://github.com/MetaMask/metamask-extension/issues/14302

## Screenshots/Screencaps

### Before
Phishing detect

<img width="839" alt="Screen Shot 2022-03-31 at 10 50 27 AM" src="https://user-images.githubusercontent.com/8112138/161120357-beae2120-c732-4e8c-a770-33adb890aece.png">

### After
Phishing detect
<img width="1440" alt="Screen Shot 2022-03-31 at 10 50 53 AM" src="https://user-images.githubusercontent.com/8112138/161120426-3f54fa5f-0769-447e-9853-1ed9241400da.png">


About page

<img width="972" alt="Screen Shot 2022-03-31 at 10 57 56 AM" src="https://user-images.githubusercontent.com/8112138/161119921-d63686ca-99cf-43a6-a223-715e5cb8fc55.png">
<img width="1009" alt="Screen Shot 2022-03-31 at 10 58 04 AM" src="https://user-images.githubusercontent.com/8112138/161119923-aa2f27f6-04f8-454f-8fbc-3ce71ef7b856.png">

Story 

<img width="1440" alt="Screen Shot 2022-03-31 at 10 48 10 AM" src="https://user-images.githubusercontent.com/8112138/161119735-dcdcb417-e903-4a6c-bb17-fe3be12ddc7b.png">
<img width="1440" alt="Screen Shot 2022-03-31 at 10 48 36 AM" src="https://user-images.githubusercontent.com/8112138/161119738-f6b20546-804c-4b46-be10-104e494ffff9.png">


## Manual testing steps

Local build
- Go to Settings > About > See logo at bottom of page

Storybook
- Go to latest build of storybook in this PR and search `InfoTab`
